lib/core/src/lib/form/components/widgets/core/form-field.model.ts

```diff
@@ -335,16 +348,30 @@ export class FormFieldModel extends FormWidgetModel {
          This is needed due to Activiti displaying/editing dates in d-M-YYYY format
          but storing on server in ISO8601 format (i.e. 2013-02-04T22:44:30.652Z)
          */
-        if (this.isDateField(json) || this.isDateTimeField(json)) {
+        if (this.isDateTimeField(json)) {
             if (value) {
                 let dateValue;
                 if (isNumberValue(value)) {
-                    dateValue = moment(value);
+                    dateValue = new Date(value);
                 } else {
-                    dateValue = this.isDateTimeField(json) ? moment.utc(value, 'YYYY-MM-DD hh:mm A') : moment.utc(value.split('T')[0], 'YYYY-M-D');
```

Comment:
is the UTC still preserved here?

Reply:
updated code to use parseISO() to preserve UTC info.